docs/python/tutorial-flask.md

```diff
@@ -502,11 +470,11 @@ You can run the app at this point, but because you haven't made use of the base
 
 Because the three pages you create in the next section extend `layout.html`, it saves time to create a **code snippet** to initialize a new template file with the appropriate reference to the base template. A code snippet provides a consistent piece of code from a single source, which avoids errors that can creep in when using copy-paste from existing code.
 
-1. In VS Code, select the **File** (Windows/Linux) or **Code** (macOS), menu, then select **Preferences** > **User snippets**.
+1. In VS Code, select **File** >  **Preferences** > **Configure User snippets**.
```

Comment:
Any reason you removed this? Mac users have **Preferences** under **Code** and they often get confused when we only mention the **File** menu

Reply:
Put back mention of Code on macOS